jobs/payment-jobs/utils/logger.py

```diff
@@ -26,6 +26,6 @@ def setup_logging(conf):
 
     if conf and path.isfile(conf):
         logging.config.fileConfig(conf)
```

Comment:
This is called before the alembic migrations, I added in a line to call it again after the migrations. That way the loggers get restored. 

Normally payment jobs doesn't run migrations.